glossary.md

```diff
@@ -0,0 +1,22 @@
+# Glossary
+
+## Application
+
+A process tree launched inside a [container](#container).
+
+## Bundle
+
+A [directory structure](bundle.md) that is written ahead of time, distributed, and used to seed the runtime for creating [containers](#container) and launching [applications](#application).
```

Comment:
On Sat, Aug 29, 2015 at 10:25:27AM -0700, Lai Jiangshan wrote:
> > ## Create
> > Creates the container: file system, namespaces, cgroups, capabilities.
>
> is some kinds of misleading. Only the network namespace can be
> created graciously before starting.

That may be.  But I think this is getting too far afield for this PR
(maybe shift it to [1]?).  I was only quoting that *existing spec*
(not suggesting we add that wording in this PR) to show that the
current spec seems to lean both ways (depending on which section
you're reading) about whether or not there will be multiple
applications in one container [2].  And if we accept that the
singular/plural application question is currently up in the air, the
‘applications’ plural suggested by this PR isn't a major concession.

But that cuts both ways, so I guess I'll just change to singular here
and isolate the singular/plural lifecycle discussion in [1].  Done in
the just pushed 5fc03cf→9db5fbe, which also rebases onto the current
master (not that there were any conflicts, but having a shorter
fork/merge delay makes for an easier-to-read history graph ;).

[1]: https://groups.google.com/a/opencontainers.org/forum/#!topic/dev/kPWu-ZavQjs
[2]: https://github.com/opencontainers/specs/pull/107/files#r38259905